packages/rs-dapi-client/src/platform/get_consensus_params.rs

```diff
@@ -0,0 +1,99 @@
+//! `GetConsensusParams` request.
+
+use dapi_grpc::platform::v0::{self as platform_proto};
+
+use crate::{transport::TransportRequest, DapiRequest, Settings};
+
+use super::IncompleteMessage;
+
+/// Request consensus params.
+#[derive(Debug)]
+pub struct GetConsensusParams {
+    /// Block height
+    pub height: u32,
```

Comment:
Isn't it optional? I guess it must be u64

Reply:
In proto it's `i32`, non optional (I'm targeting Lukasz' branch, that might be changed from dev), and since it's 32 bits and I expect no height to be negative I used `u32` to have less space for client errors